dummy PR

HCasler opened the pull request.

FNALbuild commented:
Hi @HCasler,
You have proposed changes to files in these packages:
- JobConfig

which require these tests: build.

@Mu2e/mu2eii_sm21_ci, @Mu2e/write have access to CI actions on [Mu2eII_SM21](/Mu2e/Offline/tree/Mu2eII_SM21).





:hourglass: The following tests have been triggered for 1274b23e2451905f2d19bd12eed181b705fb0cf4: build  (Build queue has 2 jobs)


[About FNALbuild.](https://mu2ewiki.fnal.gov/wiki/GitHubWorkflow#GitHub_Pull_Request_Procedures_and_FNALbuild) [Code review on Mu2e/Offline.](https://mu2ewiki.fnal.gov/wiki/GitHubWorkflow#Code_Review)

FNALbuild commented:
:sunny: The tests passed at 1274b23e2451905f2d19bd12eed181b705fb0cf4.


| Test          | Result        | Details |
| ------------- |:-------------:| ------- |
| merge | :white_check_mark: | Merged 1274b23e2451905f2d19bd12eed181b705fb0cf4 at c92e9594deadf710c9712d4a6bbed30e2fdbd4eb |
| build (prof) | :white_check_mark: | [Log file](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/scons.log). Build time: 13 min 32 sec |
| ceSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/ceSimReco.log)  |
| g4test_03MT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/g4test_03MT.log)  |
| transportOnly | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/transportOnly.log)  |
| POT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/POT.log)  |
| g4study | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/g4study.log)  |
| cosmicSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/cosmicSimReco.log)  |
| cosmicOffSpill | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/cosmicOffSpill.log)  |
| ceSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/ceSteps.log)  |
| ceDigi | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/ceDigi.log)  |
| muDauSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/muDauSteps.log)  |
| ceMix | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/ceMix.log)  |
| rootOverlaps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/rootOverlaps.log)  |
| g4surfaceCheck | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/g4surfaceCheck.log)  |
| FIXME, TODO | :white_check_mark: | [TODO (0) FIXME (0) in 0 files](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/fixme_todo.log) |
| clang-tidy | :white_check_mark: | [0 errors 0 warnings](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/artifact/clang-tidy.log) |


N.B. These results were obtained from a build of this Pull Request at 1274b23e2451905f2d19bd12eed181b705fb0cf4 after being merged into the base branch at c92e9594deadf710c9712d4a6bbed30e2fdbd4eb.


For more information, please check the job page [here](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//34/console).
Build artifacts are deleted after 5 days. If this is not desired, select `Keep this build forever` on the job page.

kutschke commented:
Test that new CI will automagically test against the correct branch of Offline

@FNALbuild run build test

FNALbuild commented:
:hourglass: The following tests have been triggered for 1274b23e2451905f2d19bd12eed181b705fb0cf4: build  (Build queue has 1 jobs)

HCasler commented:
From Jenkins log, looks like we're properly building with Mu2eII here:

[Mon Sep 27 14:15:25 CDT 2021] setup Mu2e/Production
[2021-09-27 14:15:26,085] Mu2eCI:INFO  |  API Rate Limit: 4651/5000, Reset in 1638 sec i.e. at 2021-09-27 14:42:39
[2021-09-27 14:15:26,632] Mu2eCI:INFO  |  Mu2e/Production PR 100 base ref is Mu2eII_SM21; the commit sha is c92e9594deadf710c9712d4a6bbed30e2fdbd4eb
Cloning into 'Offline'...
From https://github.com/Mu2e/Offline
 * branch            Mu2eII_SM21 -> FETCH_HEAD
Switched to a new branch 'Mu2eII_SM21'
Branch Mu2eII_SM21 set up to track remote branch Mu2eII_SM21 from origin.
Cloning into 'Production'...
From https://github.com/Mu2e/Production
 * branch            Mu2eII_SM21 -> FETCH_HEAD
Switched to a new branch 'Mu2eII_SM21'
Branch Mu2eII_SM21 set up to track remote branch Mu2eII_SM21 from origin.
From https://github.com/Mu2e/Production
 * [new ref]         refs/pull/100/head -> pr100

kutschke commented:
Thanks Helenka,

  I think there is one last base to cover – I reopened PR 99 in Production and ran it’s test.  This is to check that it correctly tests using Offline/main.

  After that I think we are done.

Rob

From: Helenka Casler ***@***.***>
Reply-To: Mu2e/Production ***@***.***>
Date: Monday, September 27, 2021 at 2:29 PM
To: Mu2e/Production ***@***.***>
Cc: Rob Kutschke ***@***.***>, Team mention ***@***.***>
Subject: Re: [Mu2e/Production] dummy PR (#100)


From Jenkins log, looks like we're properly building with Mu2eII here:

[Mon Sep 27 14:15:25 CDT 2021] setup Mu2e/Production
[2021-09-27 14:15:26,085] Mu2eCI:INFO | API Rate Limit: 4651/5000, Reset in 1638 sec i.e. at 2021-09-27 14:42:39
[2021-09-27 14:15:26,632] Mu2eCI:INFO | Mu2e/Production PR 100 base ref is Mu2eII_SM21; the commit sha is c92e959<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Mu2e_Production_commit_c92e9594deadf710c9712d4a6bbed30e2fdbd4eb&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=Zi5IH-k2_JiK_Wipiv3pwwSXO9UNaHiLP8sisfPz__k&m=Wp1fL149ZY_ksog3wGOgVpfChU4CFvE87F5WUNMDjjI&s=HKnz4outf2knaRIp1u099Zs2HX1e64je1lZbMFUZQxU&e=>
Cloning into 'Offline'...
From https://github.com/Mu2e/Offline<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Mu2e_Offline&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=Zi5IH-k2_JiK_Wipiv3pwwSXO9UNaHiLP8sisfPz__k&m=Wp1fL149ZY_ksog3wGOgVpfChU4CFvE87F5WUNMDjjI&s=ZyBKgC_fZJL3U6uajn1Cm5lP0ZUjBkwbt_Qu4AuvbKw&e=>
·         branch Mu2eII_SM21 -> FETCH_HEAD
Switched to a new branch 'Mu2eII_SM21'
Branch Mu2eII_SM21 set up to track remote branch Mu2eII_SM21 from origin.
Cloning into 'Production'...
From https://github.com/Mu2e/Production<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Mu2e_Production&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=Zi5IH-k2_JiK_Wipiv3pwwSXO9UNaHiLP8sisfPz__k&m=Wp1fL149ZY_ksog3wGOgVpfChU4CFvE87F5WUNMDjjI&s=Gi5xNcrgOna6J5OPY9GT2NVRDY-Rg1ECck7fbcBYa0A&e=>
·         branch Mu2eII_SM21 -> FETCH_HEAD
Switched to a new branch 'Mu2eII_SM21'
Branch Mu2eII_SM21 set up to track remote branch Mu2eII_SM21 from origin.
From https://github.com/Mu2e/Production<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Mu2e_Production&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=Zi5IH-k2_JiK_Wipiv3pwwSXO9UNaHiLP8sisfPz__k&m=Wp1fL149ZY_ksog3wGOgVpfChU4CFvE87F5WUNMDjjI&s=Gi5xNcrgOna6J5OPY9GT2NVRDY-Rg1ECck7fbcBYa0A&e=>
·         [new ref] refs/pull/100/head -> pr100

—
You are receiving this because you are on a team that was mentioned.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Mu2e_Production_pull_100-23issuecomment-2D928203967&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=Zi5IH-k2_JiK_Wipiv3pwwSXO9UNaHiLP8sisfPz__k&m=Wp1fL149ZY_ksog3wGOgVpfChU4CFvE87F5WUNMDjjI&s=r2Sf5GKR6FkTyOq0_ckYr6rsLYxkQzkfRC3BY6kFYHM&e=>, or unsubscribe<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ABHY5ZW6KDHSJCMV4IS42D3UEDAYHANCNFSM5EPU77DQ&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=Zi5IH-k2_JiK_Wipiv3pwwSXO9UNaHiLP8sisfPz__k&m=Wp1fL149ZY_ksog3wGOgVpfChU4CFvE87F5WUNMDjjI&s=ot6AnGeL4i2dSfMFWMUf-aj-_IaIZy6JeiGO4QD-rX0&e=>.
Triage notifications on the go with GitHub Mobile for iOS<https://urldefense.proofpoint.com/v2/url?u=https-3A__apps.apple.com_app_apple-2Dstore_id1477376905-3Fct-3Dnotification-2Demail-26mt-3D8-26pt-3D524675&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=Zi5IH-k2_JiK_Wipiv3pwwSXO9UNaHiLP8sisfPz__k&m=Wp1fL149ZY_ksog3wGOgVpfChU4CFvE87F5WUNMDjjI&s=FGiPKMdsS6cKvbCzI6kSUbewHHrwd7m9ltyiIYVQt4I&e=> or Android<https://urldefense.proofpoint.com/v2/url?u=https-3A__play.google.com_store_apps_details-3Fid-3Dcom.github.android-26referrer-3Dutm-5Fcampaign-253Dnotification-2Demail-2526utm-5Fmedium-253Demail-2526utm-5Fsource-253Dgithub&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=Zi5IH-k2_JiK_Wipiv3pwwSXO9UNaHiLP8sisfPz__k&m=Wp1fL149ZY_ksog3wGOgVpfChU4CFvE87F5WUNMDjjI&s=dTH1Y3trFgAHpVvD_QrJ6BCYeSbHThu9rTsCoWPLrCE&e=>.

FNALbuild commented:
:sunny: The tests passed at 1274b23e2451905f2d19bd12eed181b705fb0cf4.


| Test          | Result        | Details |
| ------------- |:-------------:| ------- |
| merge | :white_check_mark: | Merged 1274b23e2451905f2d19bd12eed181b705fb0cf4 at c92e9594deadf710c9712d4a6bbed30e2fdbd4eb |
| build (prof) | :white_check_mark: | [Log file](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/scons.log). Build time: 15 min 45 sec |
| ceSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/ceSimReco.log)  |
| g4test_03MT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/g4test_03MT.log)  |
| transportOnly | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/transportOnly.log)  |
| POT | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/POT.log)  |
| g4study | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/g4study.log)  |
| cosmicSimReco | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/cosmicSimReco.log)  |
| cosmicOffSpill | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/cosmicOffSpill.log)  |
| ceSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/ceSteps.log)  |
| ceDigi | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/ceDigi.log)  |
| muDauSteps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/muDauSteps.log)  |
| ceMix | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/ceMix.log)  |
| rootOverlaps | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/rootOverlaps.log)  |
| g4surfaceCheck | :white_check_mark: | [Log file.](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/g4surfaceCheck.log)  |
| FIXME, TODO | :white_check_mark: | [TODO (0) FIXME (0) in 0 files](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/fixme_todo.log) |
| clang-tidy | :white_check_mark: | [0 errors 0 warnings](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/artifact/clang-tidy.log) |


N.B. These results were obtained from a build of this Pull Request at 1274b23e2451905f2d19bd12eed181b705fb0cf4 after being merged into the base branch at c92e9594deadf710c9712d4a6bbed30e2fdbd4eb.


For more information, please check the job page [here](https://buildmaster.fnal.gov/buildmaster/job/GitHubPRTests/job/mu2e-production-build-test//37/console).
Build artifacts are deleted after 5 days. If this is not desired, select `Keep this build forever` on the job page.
